corehq/apps/zapier/views.py

```diff
@@ -86,3 +90,71 @@ def post(self, request, *args, **kwargs):
             return HttpResponse(status=400)
         delete_subscription_with_url(url)
         return HttpResponse('OK')
+
+
+class ZapierCreateCase(View):
+
+    urlname = 'zapier_create_case'
+
+    @method_decorator(csrf_exempt)
+    def dispatch(self, *args, **kwargs):
+        return super(ZapierCreateCase, self).dispatch(*args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        domain = request.GET.get('domain')
+        case_type = request.GET.get('case_type')
+        owner_id = request.GET.get('user_id')
+        properties = json.loads(request.body)
+        case_name = properties['case_name']
+        user_name = request.GET.get('user')
+
+        user = CommCareUser.get_by_username(user_name, domain)
+        couch_user = CouchUser.from_django_user(user)
+        if not couch_user.is_member_of(domain):
+            return HttpResponseForbidden("This user does not have access to this domain.")
+
+        del properties['case_name']
+
+        factory = CaseFactory(domain=domain)
+        factory.create_case(
+            case_type=case_type,
+            owner_id=owner_id,
+            case_name=case_name,
+            update=properties
```

Comment:
I think it might be good to validate that all of these properties (case_type, owner_id, case_name) are set, and return an error if they aren't.